src/Layer/babylon.glowLayer.ts

```diff
@@ -115,7 +119,7 @@
          * @param scene The scene to use the layer in
          * @param options Sets of none mandatory options to use with the layer (see IGlowLayerOptions for more information)
          */
-        constructor(public name: string, scene: Scene, options?: Partial<IGlowLayerOptions>) {
+        constructor(name: string, scene: Scene, options?: Partial<IGlowLayerOptions>) {

```

Comment:
This is a breaking change
You must comment it directly in the constructor

Reply:
Don't understand the breaking change :(
As we have "public name: string" in BABYLON.EffectLayer

Reply:
Right I missed it :)